rfcs/0124-worker-manager.md

```diff
@@ -0,0 +1,271 @@
+# RFC 124 - Worker Manager
+* Comments: [#124](https://api.github.com/repos/taskcluster/taskcluster-rfcs/issues/124)
+* Proposed by: @jhford
+
+# Summary
+Taskcluster was originally designed with a single Queue instance in the world.
+This single Queue would be used to coordinate all tasks from all Taskcluster
```

Comment:
Perhaps mention that capitalized "Queue" is what you are calling the "taskcluster-queue"?

Reply:
I added a link to the queue which I think should clarify this?